pom.xml

```diff
@@ -226,6 +226,7 @@ under the License.
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-invoker-plugin</artifactId>
+              <version>3.4.0</version>
```

Comment:
Please add TODO, with information that should verified with next parent upgrades

Reply:
TODO added.